CHANGELOG.md

```diff
@@ -20,6 +45,7 @@ These should not affect the functionality of the module.
 
 **Closed issues:**
 
+- cron.hourly/logrotate not being created [\#160](https://github.com/voxpupuli/puppet-logrotate/issues/160)
```

Comment:
I've added `skip-changelog` to this issue.

Reply:
updated